tsdb/head.go

```diff
@@ -61,18 +61,21 @@ var (
 
 // Head handles reads and writes of time series data within a time window.
 type Head struct {
-	chunkRange int64
+	// Keep all 64bit atomically accessed variables at the top of this struct.
+	// Otherwise ARM builds of Prometheus can fail due to alignment issues.
```

Comment:
This line can be removed. ARM isn't related.

Reply:
:+1: